documentation/feed-using-hadoop-pig-oozie.html

```diff
@@ -448,38 +448,32 @@ <h3 id="efficiency">Efficiency</h3>
 a number of mappers or reducers will be used to cover the file if the input file is sufficiently large.
 As each mapper will have their own client feeding to the endpoint, each with multiple threads,
 make sure the cluster can take the load.
-Each client will throttle to not overload the endpoint.
-Using a high number of effective thread (tasks times threads per mapper),
-can easily saturate feeding endpoints.
+Each client will throttle to not overload the endpoint but clients does not know how many total client there are so one can easily over saturate 
+feeding endpoints.
 </p><p>
 There are two primary factors controlling feed load: the number of mappers or
-reducers, and the number of concurrent connections per mapper/reducer.
-Control the number of splits and thus number of mappers to make based on the input file size:
-<pre>
-SET mapreduce.input.fileinputformat.split.minsize 128*1024*1024
+reducers, and the number of concurrent connections per mapper/reducer. For large jobs with lots of data one might consider 
+control the number of splits and thus number of mappers to make based on the input file size. This can be done by 
```

Comment:
For large jobs with lots of data, one might consider control*ling* the number of splits and thus number of mappers to make based on the input file size.